src/apiCalls.tsx

```diff
@@ -10,5 +11,20 @@ function fetchTea() {
             return data
         })
 }
+*/
+
+const fetchTea = async () => {
+    try {
+        const response = await fetch('https://boonakitea.cyclic.app/api/all');
+        if (!response.ok) {
+            throw new Error('failed to fetch tea data')
+        }
+        return await response.json();
+    } catch (error: any) {
+        // console.log('error', error)
+        console.log('error code', error.status)
```

Comment:
I assume you will eventually do something with the error aside from console logging it? The user needs to see some messaging too!